README.rst

```diff
@@ -28,3 +28,7 @@ provider) where the strings can be translated.
 
 Compile translated strings by running `make compile_translations`. This will produce a .mo file for each .po in the repo.
 The .mo files are read by Django and are used to provide translations in the running application.
+
+Testing credentials-themes changes locally with your credentials repo
+---------------------------------------------------------------------
+https://openedx.atlassian.net/wiki/spaces/SOL/pages/608698737/Testing+WL+themes+in+devstack
```

Comment:
I mean, since you did ask...

The header might make sense as "... changes in devstack"

Just a raw URL like that is a bit unsightly? Maybe have a sentence like "There are [instructions|URL] in the wiki." or whatever the syntax is.